pkg/pxc/app/statefulset/node.go

```diff
@@ -531,6 +537,10 @@ func (c *Node) Volumes(podSpec *api.PodSpec, cr *api.PerconaXtraDBCluster, vg ap
 			vol.Volumes = append(vol.Volumes, app.GetConfigVolumes("logcollector-config", ls["app.kubernetes.io/instance"]+"-logcollector"))
 		}
 	}
+	if cr.CompareVersionWith("1.11.0") >= 0 {
```

Comment:
Should we  create the `configmaps` and mount the dirs if  `hookScript`  sections do not exist?